tox.ini

```diff
@@ -7,6 +7,7 @@ inline-quotes = double
 envlist = py27,py36
 
 [testenv]
+install_command = pip install {opts} {packages} --only-binary rasterio 
 commands=python -m pytest --cov raster_tester --ignore=venv
 deps=
     numpy
```

Comment:
Tox makes for some very tidy test configurations. We should consider using it everywhere, even on Travis-tested repos.

Reply:
true, but I still don't really understand why I had to add `--only-binary rasterio ` while it seems to work fine in other repo